tembo-operator/src/snapshots/volumesnapshots.rs

```diff
@@ -291,7 +338,7 @@ fn generate_volume_snapshot(
                 volume_snapshot_content_name: Some(volume_snapshot_content_name.to_string()),
                 ..VolumeSnapshotSource::default()
             },
-            volume_snapshot_class_name: Some(volume_snapshot_class_name.to_string()),
```

Comment:
This is removed as it's not needed in the VolumeSnapshot and will be populated and assumed from the VolumeSnapshotContents